### ISSUE TITLE
missing "call" in add_custom_command of FindNanopb.cmake for MSVC

### ISSUE THREAD

**Issue by philippe44** [nanopb/nanopb#873]
I think the custom_command of "FindNanopb.cmake" are missing a "call" prefix that causes them to never return properly in MSVC. As a result, .pb.c and .pb.h are not built. I suggest adding something like that
```
add_custom_command(
      OUTPUT "${CMAKE_CURRENT_BINARY_DIR}/${FIL_PATH_REL}/${FIL_WE}.pb.c"
             "${CMAKE_CURRENT_BINARY_DIR}/${FIL_PATH_REL}/${FIL_WE}.pb.h"
      COMMAND ${CALL_PREFIX} ${PROTOBUF_PROTOC_EXECUTABLE}
      ARGS -I${GENERATOR_PATH} -I${GENERATOR_CORE_DIR}
           -I${CMAKE_CURRENT_BINARY_DIR} ${_nanopb_include_path}
           --plugin=protoc-gen-nanopb=${NANOPB_GENERATOR_PLUGIN}
           ${NANOPB_OPT_STRING}
           ${ABS_FIL}
      DEPENDS ${ABS_FIL} ${GENERATOR_CORE_PYTHON_SRC}
           ${ABS_OPT_FIL} ${NANOPB_DEPENDS}
      COMMENT "Running C++ protocol buffer compiler using nanopb plugin on ${FIL}"
      VERBATIM )
```
Where CALL_PREFIX is defined to "call" under MSVC and nothing otherwise. It needs to be done there as well
```
  foreach(FIL ${GENERATOR_CORE_SRC})
      get_filename_component(ABS_FIL ${FIL} ABSOLUTE)
      get_filename_component(FIL_WE ${FIL} NAME_WE)

      set(output "${GENERATOR_CORE_DIR}/${FIL_WE}_pb2.py")
      set(GENERATOR_CORE_PYTHON_SRC ${GENERATOR_CORE_PYTHON_SRC} ${output})
      add_custom_command(
        OUTPUT ${output}
        COMMAND ${CALL_PREFIX} ${PROTOBUF_PROTOC_EXECUTABLE}
        ARGS -I${GENERATOR_PATH}/proto
          --python_out=${GENERATOR_CORE_DIR} ${ABS_FIL}
        DEPENDS ${ABS_FIL}
        VERBATIM)
  endforeach()
```
I'm not submitting a PR because I'm sure the maintainers will have a better option

**Comment by PetteriAimonen** [comment]
Thanks for the report.

Unfortunately I don't have the time or experience to maintain the CMake rules either..

**Comment by philippe44** [comment]
I can offer a PR, but it might not be the most elegant 😄

**Comment by PetteriAimonen** [comment]
What this would really need is a github workflow that tests it.

**Comment by PetteriAimonen** [comment]
Fix published in nanopb-0.4.8.
